tech/languages/c/about.md

```diff
@@ -4,5 +4,67 @@ page: c
 section: tech-languages
 ---
 
-#C and C++
-This content needs to be created...
+## C and C++ installation and usage
+
+### C
+
+C is a compiled language and as such it needs a compiler. To install a compiler, simply type:
+
+```
+# sudo dnf install gcc
+```
+
+This command should download and install package with compiler called gcc.
+To compile a program written in C, simply run the gcc like this:
```

Comment:
To compile and link... (compilation only mode can be enabled by the -c option of gcc)